tests/RestfulSimpleJsonTestCase.test

```diff
@@ -81,6 +81,23 @@ class RestfulSimpleJsonTestCase extends RestfulCurlBaseTestCase {
     $this->assertEqual($decoded_json['count'], 2, 'Count was populated correctly.');
     $this->assertEqual(count($decoded_json['data']), 1, 'The correct number of items was listed.');
     $this->assertNotNull($decoded_json['next'], '"Next" property added.');
+
+    // Test the pagination links for bigger ranges.
+    $handler->setRequest(Request::create('/api/v1.5/articles', array(
+      'range' => 3,
```

Comment:
Maybe change this to `1000` just so it's clear we're passing the allowed values?